rules/declaration-block-properties-flexible-order/index.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+const stylelint = require('stylelint');
+const _ = require('lodash');
+const utils = require('../../utils');
+const declarationBlockPropertiesSpecifiedOrder = require('../declaration-block-properties-specified-order');
+
+const ruleName = utils.namespace('declaration-block-properties-flexible-order');
+
+const messages = stylelint.utils.ruleMessages(ruleName, {
+	expected: (first, second) => `Expected ${first} to come before ${second}`,
+});
+
+function createExpectedOrder(input) {
```

Comment:
This function will generate correct expected order for both rules. Because both rules can accept an array of strings as an option.